### PR TITLE
Dockerfile and docker composer added

### PR DESCRIPTION
**Pull request by anjana-somathilake**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM frolvlad/alpine-java:jre8-slim
+EXPOSE 8000
+COPY target/covid19-1.0.0-SNAPSHOT/*.* ./
+ENTRYPOINT ["java","-jar","covid19.jar"]
+
```
Comment (nilesh93) [inline Dockerfile]: Hi @anjana-somathilake 

Can't we do the build too inside the docker file?

**Comment by nilesh93** [comment]
@anjana-somathilake 
Can't we do the build too inside the Dockerfile? This Dockerfile requires the jar to be prebuilt isn't it

File: Dockerfile (hunk 1 above)
Comment (anjana-somathilake) [reply Dockerfile]: we can,  then the docker should have maven installed as a dependency, building on the dev and deploying on the server is what makes sense for developers.

Comment (anjana-somathilake) [reply Dockerfile]: do we have a CI service?

File: readme.md [hunk 2]
```diff
@@ -33,6 +33,14 @@ and add the private key from Firebase Admin SDK.
 - run `java -jar covid19-1.0.0-SNAPSHOT.jar`. Check if the jar version matches.
 - NOTE: access logs will get stored in the tomcat folder (tomcat/access/).
  These logs can be disabled from the application.yml
+ 
+### with Docker
+- build project using `mvn clean install`
+- unzip the `target/covid19-1.0.0-SNAPSHOT.zip` to `target/covid19-1.0.0-SNAPSHOT`
+- modify the db url in the application.yml (replace localhose with container name) as below  (FIXME: automate with spring profiles)
```
Comment (azhamn) [inline readme.md]: great suggestion! are you able to make this change? do not forget to create a issue :)

Comment (anjana-somathilake) [reply readme.md]: https://github.com/ICTASL/COVID-19/issues/97

**Pull request merged**